### PR TITLE
BB-63 Missing readiness probe routes

### PR DESCRIPTION
**Pull request by nicolas2bert**

### CODE REVIEW

**Review by miniscruff — APPROVED** [review]

File: bin/queuePopulator.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ async.waterfall([
         if (probeServer !== undefined) {
             probeServer.addHandler(
                 DEFAULT_LIVE_ROUTE,
+                DEFAULT_READY_ROUTE,
```
Comment (naren-rajendran) [inline bin/queuePopulator.js]:
```suggestion
                [DEFAULT_LIVE_ROUTE, DEFAULT_READY_ROUTE],
```

File: extensions/replication/replicationStatusProcessor/task.js [hunk 2]
```diff
@@ -47,6 +47,7 @@ function initAndStart() {
                 if (probeServer !== undefined) {
                     probeServer.addHandler(
                         DEFAULT_LIVE_ROUTE,
```
Comment (naren-rajendran) [inline extensions/replication/replicationStatusProcessor/task.js]:
```suggestion
                        [DEFAULT_LIVE_ROUTE, DEFAULT_READY_ROUTE],
```

**Review by naren-rajendran — CHANGES_REQUESTED** [review]
ProbeServer.addHandler takes only two arguments

**Review by naren-rajendran — APPROVED** [review]

**Comment by nicolas2bert** [comment]
@bert-e reset

**Comment by nicolas2bert** [comment]
@bert-e approve

**Pull request merged**